src/main/resources/db/migration/V1.0.2__init_tables.sql

```diff
@@ -0,0 +1,15 @@
+SET
+search_path TO wa_workflow_api;
+
+DROP TABLE IF EXISTS idempotent_keys;
+
+CREATE TABLE idempotent_keys
```

Comment:
Maybe as a future ticket we could put a comment on the table/columns to describe what they are?